### PR TITLE
Migrated to ESLint.

### PR DESCRIPTION
**Pull request by cecobask**
Migrated the project from tslint to eslint.

### CODE REVIEW

File: packages/graphback-cli/src/helpers/transformOpenApiSpec.ts [hunk 1]
```diff
@@ -79,3 +45,37 @@ async function processSingleDefinition(model: string, isYaml: boolean, openApiCo
...
     }
 }
+
+/**
+ * OpenAPI command line tool
+ * 1. Reads all the OpenAPI yaml and json files from the model folder
+ * 2. Generates GraphQL Schema
+ * 3. Renames original OpenAPI definitions so they will not be processed anymore
+ */
+export const transformOpenApiSpec = async () => {
```
Comment (wtrocki) [inline packages/graphback-cli/src/helpers/transformOpenApiSpec.ts]: Was this some kind of rule that was failing? Order of functions

Comment (cecobask) [reply packages/graphback-cli/src/helpers/transformOpenApiSpec.ts]: Eslint complained that the function shouldn't be used before it is defined

**Comment by wtrocki** [comment]
Thank you so much for contribution. I needed to change base as we have large portion of changes of separate branch

**Comment by wtrocki** [comment]
Closing for #664

**Pull request closed without merging**